pkg/markdown/markdown.go

```diff
@@ -0,0 +1,38 @@
+package markdown
+
+import (
+	"os"
+	"strings"
+
+	"github.com/charmbracelet/glamour"
+)
+
+func Render(text string, bgColor string) (string, error) {
```

Comment:
I find it a bit confusing that this method takes an explicit argument to declare the background color, but that this value is potentially overridden by the GLAMOUR_STYLE environment variable.

If we're passing in an explicit argument, I feel like it should be the resolved style in the first place.